### ISSUE TITLE
COM Group or Resource Error

### ISSUE THREAD

**Issue by BobbyShoe** [MicrosoftEdge/WebView2Feedback#1010]
On one specific test machine we are receiving the following error when trying to load a WebView2 instance.  Our application is working fine on multiple other machines.

System.Runtime.InteropServices.COMException (0x8007139F): The group or resource is not in the correct state to perform the requested operation. (0x8007139F)
   at System.Runtime.InteropServices.Marshal.ThrowExceptionForHR(Int32 errorCode)
   at Microsoft.Web.WebView2.Core.CoreWebView2Environment.CreateCoreWebView2ControllerAsync(IntPtr ParentWindow)

We are calling this from a 
- C# .Net 5 application
- using WebView2 NuGet package version 1.0.705.50
- using Fixed WebView2 runtime version 88.0.705.74.x64
- Windows 10 OS

We tried awaiting the EnsureCoreWebView2Async with explicit environments, and using the CoreWebView2InitializationCompleted event method and letting the environment initialize by setting the Source on the WebView2 class, but received the same error both times on the machine.

Is there anyway to get a more detailed error message?  Any other suggestions on something else to try?

Thanks,
Bobby

[AB#31949034](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/31949034)

**Comment by champnic** [comment]
Hey @BobbyShoe - Sorry you're running into this. I'll open this bug on our backlog to take a look. Can you try with the latest prerelease package (1.0.790-prerelease) and see if the issue still repros?

**Comment by BobbyShoe** [comment]
@champnic Kind of an odd conclusion to this issue.  We upgraded to the latest pre-release version, reinstalled our software and the problem still existed initially on the one machine.  The user went back today (a few days after last testing) to see if they could find more information in some log files, and the WebView2 control loaded fine.  Not sure what may have changed, but it is working now.

Closing for now, hopefully the issue does not return.  Thanks for the help.
